### PR TITLE
Renamed mapwindow to fullscreen

### LINKED ISSUE #105: Fix and rename mapwindow control
I suggest we rename the mapwindow control to the more descriptive name fullscreen. The control also needs some updates to follow the Origo guide lines.

### PR DESCRIPTION
**Pull request by afogelberg**
Fixes #105

### CODE REVIEW

**Comment by markus659** [comment]
It's good to fix some of these lingering issues. I will take a look at the changes later today.

File: src/fullscreen.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+var viewer = require('./viewer');
+
+var tooltip;
+var mapTarget;
+
+function Init(opt_options) {
+  var options = opt_options || {};
+  var target = options.target || '#o-toolbar-navigation';
+  mapTarget = viewer.getTarget();
+  tooltip = options.tooltipText || "Visa stor karta";
```
Comment (markus659) [inline src/fullscreen.js]: Single quotes for strings.

**Comment by markus659** [comment]
I think we could remove the old src\mapwindow.js file? Also, the current origo.min.js might need to be rebuilt, since it's pointing to o-mapwindow-button.

**Comment by afogelberg** [comment]
@markus659 fixed the changes you requested.

**Comment by markus659** [comment]
I believe it's ready for merging now.

**Pull request merged**